charts/zabbix/README.md.gotmpl

```diff
@@ -142,6 +160,15 @@ helm uninstall zabbix -n monitoring
 
 # Breaking changes of this helm chart
 
+## Version 6.2.0
```

Comment:
Hi @fibbs!

I suggest changing to 7.0.0 version

Reply:
Still believe 6.2.0 would be the next major

Reply:
Okay @fibbs. I thought I'd put more emphasis on this break change... but we can keep the deal.